fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+const url = (path) => `${process.env.REACT_APP_BASE_URL}${path}`;
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    async function getRequest() {
+      try {
+        const response = await fetch(url`/api/todos`)
+        if (!response.ok) {
+          console.log('GET: Error');
+          throw new Error(`Response error: ${response.status}`)
+        }
+        const jsonData = await response.json();
+        setTodos(jsonData);
+        console.log('GET: Success');
+      }
+      catch (err) {
+        setError(err)
+      }
+      finally {
+        setIsLoading(false)
+      }
+    }
+    getRequest();
+    /* Use fetch to send a GET request to `/api/todos`.
+     * Once the response JSON is received and parsed,
+     * update state with the received todos.
+     * Handle errors correctly and set `error` to any error or error status code.
+     * Set `isLoading` to false when all network activity is finished.
+     */
+  }, []);
+
+  function addTodo(newTodo) {
+
+      async function addPost() {
+        try {
+          const response = await fetch(url`/api/todos`, {method : 'POST', headers: {"Content-Type" : "application/json"}, body : JSON.stringify(newTodo)})
```

Comment:
I don't believe this works. Did you verify that the TODO made it into the database?
How many arguments are you passing to `fetch`? There should be 2. How many arguments are you passing to `url`? There should be 1.